src/platform/src/Result/Metadata/TokenUsage.php

```diff
@@ -46,6 +48,7 @@ public function jsonSerialize(): array
             'prompt_tokens' => $this->promptTokens,
             'completion_tokens' => $this->completionTokens,
             'thinking_tokens' => $this->thinkingTokens,
+            'tool_tokens' => $this->toolTokens,
```

Comment:
`tool_tokens` should be added to `examples/bootstrap.php` `print_token_usage` function